src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4852classannotations/InputClassAnnotations.java

```diff
@@ -1,35 +1,44 @@
 package com.google.checkstyle.test.chapter4formatting.rule4852classannotations;
 
+/**
+ * Test class for checking the rule 4852.
+ */
```

Comment:
can we avoid such javadocs ?
this is already in name of fodler and package

Reply:
We're checking against the whole google config so as of it's rules, we need javadoc for classes, interfaces, etc